sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -294,23 +294,25 @@ class Dataset[T] private[sql](
     // We set a minimum column width at '3'
     val minimumColWidth = 3
 
+    //Regular expression matching full width characters
```

Comment:
Very small nit that might fail scalastyle -- space after comment slashes

Reply:
OK, thanks.